Docs/LinuxOsxInstall.md

```diff
@@ -22,12 +22,29 @@ In general, ensure you have SDL2, OpenGL and dotnet 6 or later. Use the `Yafc` e
 Note that [the version available on Flathub](https://flathub.org/apps/details/com.github.petebuffon.yafc) is not the Community Edition. Its repo can be found at https://github.com/petebuffon/yafc. 
 
 ## OSX installation instructions
+Below are the instructions for the arm64 Macs. To use them for Intel Macs, please apply the changes to the folder `Yafc/lib/osx` instead of `Yafc/lib/osx-arm64`.
 - [Install dotnet core (v6.0 or later)](https://dotnet.microsoft.com/download)
```

Comment:
I updated to .net core 8.0 a while back right?

Reply:
True. Will fix.

Reply:
I see that the Linux instructions also/still mention 6.0

Reply:
Updated. Please verify.